application/database/migrations/2023_03_30_143724_create_classifier_value_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use App\Enums\ClassifierValueType;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('classifier_values', function (Blueprint $table) {
+            $table->uuid('id')->primary()->default(DB::raw('gen_random_uuid()'));
+            $table->timestamps();
```

Comment:
Seems like the `deleted_at` column has timezone information, but `created_at` and `updated_at` columns do not. Better to go for a consistent approach. Laravel’s documentation [recommends always including](https://laravel.com/docs/10.x/eloquent-mutators#date-casting-and-timezones) timezone information, so I would suggest: 

```suggestion
            $table->timestampsTz();
```

Quote from docs:
> By default, the date and datetime casts will serialize dates to a UTC ISO-8601 date string (1986-05-28T21:05:54.000000Z), regardless of the timezone specified in your application's timezone configuration option. You are strongly encouraged to always use this serialization format, as well as to store your application's dates in the UTC timezone by not changing your application's timezone configuration option from its default UTC value. Consistently using the UTC timezone throughout your application will provide the maximum level of interoperability with other date manipulation libraries written in PHP and JavaScript.